### ISSUE TITLE
gem releases are running behind

### ISSUE THREAD

**Issue by peterberkenbosch** [mollie/mollie-api-ruby#140]
https://rubygems.org/gems/mollie-api-ruby is only listing up to `4.7.1` while the current tagged version here is `4.9.0`

**Comment by tolgap** [comment]
Hello maintainers. Will 4.8 and 4.9 eventually be packaged and released?

**Comment by peterberkenbosch** [comment]
Gently pinging @justincase, @vernondegoede .

**Comment by justincase** [comment]
Hi @peterberkenbosch, I'll follow up with @vernondegoede. Thanks!

**Comment by vernondegoede** [comment]
It seems that OTP has been enabled, which blocks automatic releases. I've asked @martijnpolder to look into this, after which we can trigger the pipelines again.

Thanks for the heads-up @peterberkenbosch!

**Comment by peterberkenbosch** [comment]
Awesome! Thanks all!

**Comment by peterberkenbosch** [comment]
@martijnpolder any updates on this? More then happy to jump on this on my opensource Friday cycles. So feel free to reach out if there is some additional help needed. //cc @vernondegoede

**Comment by tbolender** [comment]
It would be nice to have this issue solved to be able to use captures in our production code!

**Comment by Bramjetten** [comment]
I was surprised by this just now. Would appreciate a new gem release ✌️

**Comment by justincase** [comment]
I'll create a GitHub Release for v4.10.0 this week. However, publishing to RubyGems isn't possible at the moment because of the OTP issue mentioned by Vernon.

**Comment by justincase** [comment]
`v4.10.0` is now released on GitHub ([CHANGELOG](https://github.com/mollie/mollie-api-ruby/blob/master/CHANGELOG.md#4100---2021-11-20))

I've sent out reminders to the relevant teams at Mollie that the RubyGems releases are still an issue. In the meantime, you can use the git tag in your Gemfile. For example:

```ruby
gem 'mollie-api-ruby', git: 'https://github.com/mollie/mollie-api-ruby.git', tag: 'v4.10.0'
```

**Comment by peterberkenbosch** [comment]
great! Thanks @justincase 

You can also use the github shortcut like this:

```ruby
gem "mollie-api-ruby", github: "mollie/mollie-api-ruby", tag: "v4.10.0"
```

**Comment by justincase** [comment]
No updates at this time, sorry. I've sent a reminder on #158.

**Comment by justincase** [comment]
Tagged v4.12.0 ([CHANGELOG](https://github.com/mollie/mollie-api-ruby/blob/v4.12.0/CHANGELOG.md)):

```ruby
gem "mollie-api-ruby", github: "mollie/mollie-api-ruby", tag: "v4.12.0"
```

**Comment by frenkel** [comment]
The current release on rubygems is over 4 years old which might give users the impression it is unmaintained by Mollie. Would it be possible to push a release to rubygems.org by hand, for example once a year?

**Comment by justincase** [comment]
All the versions that were absent from RubyGems.org have been published, including the latest release, v4.13.0. Going forward, new versions are automatically published when tagged. Thanks for you patience!

**Comment by frenkel** [comment]
Great, thank you!
